src/js/base/UploadBar.tsx

```diff
@@ -90,7 +99,7 @@ export const UploadBar = ({
     });
 
     return (
-        <StyledUploadBar active={isDragAccept} {...rootProps}>
+        <UploadBarContainer active={isDragAccept} {...rootProps}>
```

Comment:
nitpick:

This usage of `styled` in a component name is actually okay!

Generally we avoid it but the exception is for cases like this where you have a top level container in a another component where it is used to style the main component. 

For non top-level components it's best to avoid styled because there is likely a much more descriptive name that can be used. In this case it is hard to really come up with a name that really describes what the component is doing other than styling the uploadBar.